app/views/admin/users/index.html.erb

```diff
@@ -1,10 +1,37 @@
-<h1><%= t(".title") %></h1>
-<% @users.each do |user| %>
-  <ul id="user_<%= user.id %>">
-    <li><%= t(".name") %>: <%= user.name %></li>
-    <li><%= t(".email") %>: <%= user.email %></li>
-    <li><%= t(".admin") %>: <%= user.admin? %></li>
-    <li><%= link_to t(".links.delete"), admin_user_path(user), method: :delete %>
-      | <%= link_to t(".links.edit"), edit_admin_user_path(user) %></li>
-  </ul>
-<% end %>
+<table>
+  <caption class="table_align"><%= t(".title") %></caption>
```

Comment:
Is the class here needed? or can you style this element by scoping it to `table caption { ... }`